modules/lambda_slack_webhook/locals.tf

```diff
@@ -1,3 +1,3 @@
 locals {
-  is_production = (terraform.workspace == "production") ? true : false
+  is_production = (terraform.workspace == "production" || terraform.workspace == "pathtolive-ci") ? true : false
```

Comment:
We use this at a couple of places, do you think it would be cleaner if we add another var for `is_path_to_live` and then use it when necessary:

`local.is_production || local.is_path_to_live  ? 1 : 0`